atomium/includes/alter/views.inc

```diff
@@ -9,14 +9,15 @@
  * Implements hook_views_pre_render().
  */
 function atomium_views_pre_render(view $view) {
+  $extend = array();
   $display = $view->display[$view->current_display];
-  $style_plugin = $view->style_plugin->definition['theme'];
 
-  $extend = array(
-    $style_plugin,
-    $view->name,
-    $display->display_plugin,
-  );
+  if (isset($view->style_plugin->definition['theme'])) {
+    $style_plugin = $view->style_plugin->definition['theme'];
```

Comment:
This line can be removed, there is no need to assign the value to a new variable, it can directly go in `$extend`.